### ISSUE TITLE
Leistungsprognose auf 3 Flächen erweitern.

### ISSUE THREAD

**Issue by Cattpot** [StrathCole/ioBroker.plenticore#70]
Hallo, wäre es möglich die Leistungsprognose auf 3 Flächen zu erweitern? Hat ja auch 3 MPPTs.. 
In meinem Fall Ost-Süd-West 3 verschiedene Ausrichtungen. 

Danke lg!

**Comment by menkej** [comment]
Ja, spannend. Um es noch exotischer zu machen: ich plane zu meiner Süd-Anlage (2 Strings, 9,6 kWp) noch zwei weitere Anlagen (einmal West, einmal Ost, je ca 10-15 kWp). Hier werden aber separate Wechselrichter fällig... Sicher kein Standardfall mehr... Vielleicht wird es auch kein Kostal, dann wäre der Überschuss aber trotzdem "im Hausnetz"...
